distribution/lib/Standard/Base/0.0.0-dev/src/Data/Ordering.enso

```diff
@@ -12,6 +12,10 @@ import project.Nothing.Nothing
 import project.Panic.Panic
 from project.Data.Boolean import Boolean, False, True
 
+# needed by Comparator_Spec:
+polyglot java import org.enso.base.ObjectComparator
+polyglot java import org.enso.base.CompareException
```

Comment:
Keep or rewrite? That the [question](https://github.com/enso-org/enso/pull/9997#discussion_r1608274630)!

Reply:
OK, I guess I leave such a refactorings to you, libs guys.

Reply:
Import moved to `Extra_Imports.enso` file in aaa5e9ec9e